lib/main_common.pm

```diff
@@ -511,7 +514,7 @@ sub load_system_role_tests {
         loadtest "installation/setup_online_repos";
     }
     # Do not run on REMOTE_CONTROLLER, IPMI and on Hyper-V in GUI mode
-    if (!get_var("REMOTE_CONTROLLER") && !check_var('BACKEND', 'ipmi') && !is_hyperv_in_gui && !get_var("LIVECD")) {
+    if (!get_var("REMOTE_CONTROLLER") && !check_var('BACKEND', 'ipmi') && !is_hyperv_in_gui && !get_var("LIVECD") && !check_var('BACKEND', 'spvm')) {
```

Comment:
can you tell why the following module does not work? Is it about availability of a serial port? Btw I think the comment above is redundant (and now incomplete), better delete the comment